bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -50,19 +64,47 @@
 
     private static final String PRIMARY_ADDRESS = "primaryAddress";
     private static final String BROADCAST_ADDRESS = "broadcastAddress";
+    private static final String POLL_INTERVAL = "service.config.network.pollInterval";
```

Comment:
Still not. Please change `"service.config.network.pollInterval"` to `"pollInterval"`

Reply:
oic, sorry

Reply:
sorry i missed it.